### PR TITLE
Bugfix etb#241 sunsensor

### PR DESCRIPTION
**Pull request by jrachucki-n7s**

### CODE REVIEW

File: Drivers/seds/SunSensorExample/work/manager/implem/default/SDL/src/manager.pr [hunk 1]
```diff
@@ -0,0 +1,13 @@
+process Manager;
```
Comment (Lurkerpas) [inline Drivers/seds/SunSensorExample/work/manager/implem/default/SDL/src/manager.pr]: Why is this present?

File: Drivers/seds/AfecHwasComponent.xml [hunk 2]
```diff
@@ -427,38 +542,71 @@
...
+                              <Value value="AfecHwas_Instance_Afec1" />
+                            </Assignment>
+                            <Call activity="ReturnValAndDisableChannels" />
+                          </OnConditionTrue>
+                        </Conditional>
+                      </OnConditionFalse>
                     </Conditional>
+                  </OnConditionTrue>
+                  <OnConditionFalse>
+                    <SendCommandPrimitive shortDescription="Enable Afec interrupt" interface="InterruptManagement" command="EnableInterrupt">
```
Comment (Lurkerpas) [inline Drivers/seds/AfecHwasComponent.xml]: Is this needed?

File: Drivers/seds/SunSensorExample/work/manager/implem/default/SDL/src/manager.pr (hunk 1 above)
Comment (jrachucki-n7s) [reply Drivers/seds/SunSensorExample/work/manager/implem/default/SDL/src/manager.pr]: Deleted in new commit

File: Drivers/seds/AfecHwasComponent.xml (hunk 2 above)
Comment (jrachucki-n7s) [reply Drivers/seds/AfecHwasComponent.xml]: It is not. I have deleted it

Comment (Lurkerpas) [reply Drivers/seds/AfecHwasComponent.xml]: Have you though?

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**